modular_zubbers/code/modules/emotes/emotes.dm

```diff
@@ -109,3 +109,26 @@
 			return 'modular_zubbers/code/modules/emotes/sound/voice/male_sigh_exasperated.ogg'
 		return 'modular_zubbers/code/modules/emotes/sound/voice/female_sigh_exasperated.ogg'
 	return
+
+/datum/emote/living/tail_thump
+		key = "tailthump"
+		key_third_person = "thumps their tail!"
+		message = "thumps their tail!"
+		emote_type = EMOTE_AUDIBLE
+		vary = TRUE
+		sound = 'modular_zubbers/code/modules/emotes/sound/voice/tailthump.ogg' // See https://github.com/shiptest-ss13/Shiptest/pull/2159
+
+/datum/emote/living/tail_thump/can_run_emote(mob/user, status_check, intentional, params)
+	var/obj/item/organ/external/tail/tail = user.get_organ_slot(ORGAN_SLOT_EXTERNAL_TAIL)
+	if(tail:)
+		return ..()
+	return FALSE
```

Comment:
```suggestion
/datum/emote/living/tail_thump/can_run_emote(mob/user, status_check, intentional, params)
	var/obj/item/organ/external/tail/tail = user.get_organ_slot(ORGAN_SLOT_EXTERNAL_TAIL)
	if(tail)
		return ..()
	return FALSE
```

Reply:
Hmm, I added the colons coz build.bat was giving me an error titled 'expected :' on that line and refusing to compile.

Reply:
That's honestly kinda weird especially since the indents look fine... maybe try switching the logic around to
```
	if(isnull(tail))
		return FALSE
	return ..()
```

Reply:
Yeah it compiled like that, got no issues there! Lemme push it and it should be good to go and less horribly formatted.